src/Dev/SapphireTest.php

```diff
@@ -161,6 +161,8 @@ abstract class SapphireTest extends TestCase implements TestOnly
      */
     protected static $tempDB = null;
 
+    protected FixtureFactory|bool $fixtureFactory;
```

Comment:
This was being set as a dynamic property in setUp() via `$fixtureState->getFixtureFactory(static::class);` - getFixtureFactory() can return `FixtureFactory|bool` hence why I've used that as the type.